provider/cmd/tests/TestTasks.yml

```diff
@@ -10,6 +10,16 @@ tasks:
     env:
       AWS_PROFILE: "nuage-dev"
       AWS_DEFAULT_REGION: "eu-west-1"
+      FULL_TEST: true
+  quick:
```

Comment:
Quick should probably be the default no ?
And the test variant with database should probably be "full"

(I know it's just syntax but it kinda bothers me : what if we have another variant for some reason later on, shall we name it "quick-appsync" ?)